app/views/root/_navbar.html.erb

```diff
@@ -0,0 +1,66 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+		<a class="navbar-brand" href="/">
+      <img src="/docs/5.1/assets/brand/bootstrap-logo.svg" alt="" width="30" height="24" class="d-inline-block align-text-top">
+      Blog
+    </a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="/">Home</a>
+        </li>
+			<%if user_signed_in?%>
+				<li class="nav-item">
+					<%= link_to 'Logout', destroy_user_session_path, class:'nav-link', :method => :delete, data: {:confirm => "You sure?" }%>
+				</li>
+				<li class="nav-item">
+          <div class="nav-link" data-bs-toggle="modal" data-bs-target='#c_a_modal' >Post Article</div>
```

Comment:
indentation

Reply:
indentation is right here its is looking a lot here but actually is perfect